### PR TITLE
#1340: Code cleanup: remove profiling and --one-shot

### PR DESCRIPTION
**Pull request by jgoday**
See #1340 
1) Remove one-shot command line argument
2) Remove profile feature

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -37,8 +35,6 @@ toml = "0.5"
 libpulse-binding = { optional = true, version = "2.0", default-features = false }
 notmuch = { optional = true, version = "0.7.0" }
 maildir = { optional = true, version = "0.5" }
-# Used only in debug build for profiling blocks
-cpuprofiler = { version = "0.0.4", optional = true }
 progress = { version = "0.2", optional = true }
```
Comment (MaxVerevkin) [inline Cargo.toml:38]: Is think `progress` is safe to remove too.

**Comment by ammgws** [comment]
Thanks!

**Comment by colemickens** [comment]
I think this broke the build, it looks like Cargo.lock didn't get properly updated.

**Comment by ammgws** [comment]
Should be OK now

**Pull request merged**